src/app/layout.css

```diff
@@ -6,14 +6,18 @@
 
 .section {
   margin-bottom: 4rem;
+}
 
-  a {
-    border-bottom: 1px dotted var(--black-secondary);
-  }
+.section  a {
```

Comment:
CSS Nesting is supported above iOS 17.2 and we use a simple css style for a while.
https://caniuse.com/css-nesting